Orange/projection/pca.py

```diff
@@ -154,8 +154,7 @@ def compute(self, data, pca_space):
         return pca_space[:, self.feature]
 
 
-class RemotePCA:
-    def __new__(cls, data, batch=100, max_iter=100):
+def remotePCA(data, batch=100, max_iter=100):
```

Comment:
Please update the import in widget as well.
https://github.com/biolab/orange3/blob/master/Orange/widgets/unsupervised/owpca.py#L154